cmd/cli/plugin/feature/README.md

```diff
@@ -1,20 +1,20 @@
-# feature
+# Feature
 
-Feature plugin lets you operate on Features and FeatureGates
+Feature plugin lets you operate on features and featuregates.
```

Comment:
`Feature plugin gives access to features using featuregates` or something like that?

Reply:
I changed it to `Feature plugin gives access to feature(s) that are gated by featuregate(s).`. I'll resolve for now and you can provide another suggestion in a separate review comment.

Reply:
> Feature plugin gives access to feature(s) that are gated by featuregate(s)
This is not always true, when can list features that are not part of any featuregate

Reply:
@yharish991, what do you think of @rajathagasthya 's suggestion, `Feature plugin gives access to features using featuregates`? If you don't like that, I'll revert the changes to the orginal wording, `Feature plugin lets you operate on features and featuregates.`